Makefile

```diff
@@ -1,284 +1,25 @@
-	NPM_ARGS := --prefix $(NPM_CONFIG_PREFIX)
+ifneq ($(IN_DOCKER),)
+	SUB_MAKEFILE = Makefile-os
 endif
-
-NODE_MODULES := $(NPM_CONFIG_PREFIX)node_modules/
-STATIC_JS := static/js/node_lib/
-STATIC_JQUERY_UI := static/js/node_lib/ui/
-
-NODE_LIBS_JS := \
-less/dist/less.js \
-jquery/dist/jquery.js
-
-NODE_LIBS_JQUERY_UI := \
-jquery-ui/ui/version.js \
-jquery-ui/ui/data.js \
-jquery-ui/ui/disable-selection.js \
-jquery-ui/ui/ie.js \
-jquery-ui/ui/keycode.js \
-jquery-ui/ui/escape-selector.js \
-jquery-ui/ui/labels.js \
-jquery-ui/ui/jquery-1-7.js \
-jquery-ui/ui/plugin.js \
-jquery-ui/ui/safe-active-element.js \
-jquery-ui/ui/safe-blur.js \
-jquery-ui/ui/scroll-parent.js \
-jquery-ui/ui/focusable.js \
-jquery-ui/ui/tabbable.js \
-jquery-ui/ui/unique-id.js \
-jquery-ui/ui/position.js \
-jquery-ui/ui/widget.js \
-jquery-ui/ui/widgets/menu.js \
-jquery-ui/ui/widgets/mouse.js \
-jquery-ui/ui/widgets/autocomplete.js \
-jquery-ui/ui/widgets/datepicker.js \
-jquery-ui/ui/widgets/sortable.js
+	SUB_MAKEFILE = Makefile-docker
+include $(SUB_MAKEFILE)
 
 help:
 	@echo "Please use 'make <target>' where <target> is one of the following commands."
-	@echo "Commands that are designed be run in the container:"
-	@echo "  full_init         to init the code, the dependencies and the database"
-	@echo "  full_update       to update the code, the dependencies and the database"
-	@echo "  initialize_db     to create a new database"
-	@echo "  initialize_docker to initialize a docker image"
-	@echo "  populate_data     to populate a new database"
-	@echo "  reindex           to reindex everything in elasticsearch, for AMO"
-	@echo "  update_deps       to update the pythondependencies"
-	@echo "  update_db         to run the database migrations"
-	@echo "Commands that are designed to be run in the host:"
-	@echo "  debug             to connect to a running addons-server docker for debugging"
-	@echo "  djshell           to connect to a running addons-server docker django shell"
-	@echo "  dbshell           to connect to a running addons-server docker database shell"
-	@echo "  make              to connect to a running addons-server docker and run make ARGS"
-	@echo "  shell             to connect to a running addons-server docker shell"
-	@echo "  tdd               to run the entire test suite, but stop on the first error"
-	@echo "  test              to run the entire test suite"
-	@echo "  test_es           to run the ES tests"
-	@echo "  test_failed       to rerun the failed tests from the previous run"
-	@echo "  test_force_db     to run the entire test suite with a new database"
-	@echo "  test_no_es        to run all but the ES tests"
-	@echo "  update_docker     to update a docker image"
 	@echo "Commands that are designed to be run in either the container or the host:"
 	@echo "  docs              to builds the documentation"
 	@echo "  flake8            to run the flake8 linter"
 	@echo "  update_code       to update the git repository"
```

Comment:
Ah, I removed that one.  (I thought we had a discussion about it being redundant because no-one pulls updates from github with makefile)